lib/api/controllers/collectionController.js

```diff
@@ -90,6 +90,18 @@ class CollectionController extends BaseController {
   getMapping(request) {
     assertHasIndexAndCollection(request);
 
+    // In HTTP, booleans are flags: if it's in the querystring, it's set, whatever
+    // its value.
+    // If a user needs to unset the option, they need to remove it from the querystring.
+    if (request.context.connection.protocol !== 'http'
+      && !_.isNil(request.input.args.includeKuzzleMeta)
+      && typeof request.input.args.includeKuzzleMeta !== 'boolean') {
+
+      throw new BadRequestError(`Invalid includeKuzzleMeta value ${request.input.args.includeKuzzleMeta}: boolean expected`);
+    } else if (request.context.connection.protocol === 'http') {
+      request.input.args.includeKuzzleMeta = request.input.args.includeKuzzleMeta !== undefined ? true : false;
+    }
+    console.log(request.input.args)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a semicolon at the end of this statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)